geometry_msgs/Point point # point we want to transform
geometry_msgs/TransformStamped transform #the transform we want to use
---
geometry_msgs/Point new_point # the transformed point
